fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/Privilege.java

```diff
@@ -31,25 +34,43 @@ public enum Privilege {
     CREATE_PRIV("Create_priv", 6, "Privilege for creating database or table"),
     DROP_PRIV("Drop_priv", 7, "Privilege for dropping database or table"),
     USAGE_PRIV("Usage_priv", 8, "Privilege for using resource or workloadGroup"),
+    // in doris code < VERSION_130
     SHOW_VIEW_PRIV("Show_view_priv", 9, "Privilege for show create view"),
-    CLUSTER_USAGE_PRIV("Cluster_Usage_priv", 10, "Privilege for using cluster");
 
-    public static Privilege[] privileges = {
-            NODE_PRIV,
-            ADMIN_PRIV,
-            GRANT_PRIV,
-            SELECT_PRIV,
-            LOAD_PRIV,
-            ALTER_PRIV,
-            CREATE_PRIV,
-            DROP_PRIV,
-            USAGE_PRIV,
-            SHOW_VIEW_PRIV,
-            CLUSTER_USAGE_PRIV
-    };
+    // in cloud code < VERSION_130
+    CLUSTER_USAGE_PRIV("Cluster_Usage_priv", 9, "Privilege for using cluster"),
+    // compatible doris and cloud, and 9 ~ 11 has been contaminated
+    CLUSTER_USAGE_PRIV_COMPATIBLE("Cluster_Usage_priv", 12, "Privilege for using cluster"),
+    // 13 placeholder for stage
+    STAGE_USAGE_PRIV_COMPATIBLE("Stage_Usage_priv", 13, "Privilege for using stage"),
+    SHOW_VIEW_PRIV_COMPATIBLE("Show_view_priv", 14, "Privilege for show create view");
```

Comment:
SHOW_VIEW_PRIV_COMPATIBLE==》 SHOW_VIEW_PRIV
SHOW_VIEW_PRIV==>SHOW_VIEW_PRIV_Deprecated
Can we name it this way? This can reduce some modifications？

Reply:
fixed